.gitignore

```diff
@@ -12,3 +12,4 @@ node_modules/
 project/project
 dynamodb-local/
 project/metals.sbt
+dynamodb-local-metadata.json
```

Comment:
unrelated to scalafmt, but I noticed this file was being added when running tests locally